components/Layout/Layout.js

```diff
@@ -0,0 +1,10 @@
+import Navigation from "../Navigation";
+
+export default function Layout({ children }) {
+  return (
+    <>
+      {children}
+      <Navigation />
+    </>
+  );
+}
```

Comment:
Is this component really necessary?

Reply:
We did in one session an exercise and we had the Navigation component (renders all the nav links) wrap inside this Layout component,  i found it good because at the end just export the <Layout/> to the _app.js